superduperdb/base/datalayer.py

```diff
@@ -105,17 +105,16 @@ def __init__(
     def rebuild(self, cfg=None):
         from superduperdb.base import build
 
-        self.databackend = build.build_databackend(cfg.data_backend if cfg else None)
-        self.compute = build.build_compute(cfg.cluster.compute if cfg else None)
+        cfg = cfg or s.CFG
```

Comment:
Why not `def rebuild(self, cfg=s.CFG)` ?

Reply:
sure Thanks